core/src/lib.rs

```diff
@@ -93,6 +93,16 @@ pub fn get_api() -> &'static GodotApi {
     unsafe { GODOT_API.as_ref().expect("API not bound") }
 }
 
+#[inline]
+#[doc(hidden)]
+#[cfg(feature = "api_struct_raw_init")]
+pub unsafe fn set_api(api_raw: *const sys::godot_gdnative_core_api_struct) {
```

Comment:
Since that's a global and exported symbol I think that it's worth naming it in a way that avoids collisions. how about `gnative_rust_init_api` ?

Reply:
Would be good, I copied that code from a macro, maybe that occurance could be changed to a call to this instead?